runfolder/services.py

```diff
@@ -92,9 +92,12 @@ def _validate_is_being_monitored(self, path):
 
         :raises PathNotMonitored
         """
-        monitored = any([path.startswith(mon) for mon in self._monitored_directories()])
-        if not monitored:
-            raise PathNotMonitored("The path {0} is not being monitored".format(path))
+        monitored = list(self._monitored_directories())
+        is_monitored = any([path.startswith(mon) for mon in monitored])
```

Comment:
Does this mean that the monitoring is recursive?

I'm thinking that `startswith` would mean that if `/data/testarteria1/runfolders` is monitored then so is `/data/testarteria1/runfolders/some_subfolder`? I'm thinking that maybe we want to be stricter than that and only allow one specific directory to be monitored?